opm/output/OutputWriter.hpp

```diff
@@ -94,7 +99,8 @@ class OutputWriter {
                                 time_t current_posix_time,
                                 double seconds_elapsed,
                                 data::Solution reservoirState,
-                                data::Wells,
+                                data::Wells wellState,
+                                const std::vector<data::CellData> cellData,
```

Comment:
It turns out that this change is not needed, as this abstract class is not used anymore at all! (There is an unneeded include in opm-simulators, that is all).